From b0f54798edd95f9d1ed472bd5496ac424dce6ff4 Mon Sep 17 00:00:00 2001
From: Daniel Vetter <daniel.vetter@ffwll.ch>
Date: Wed, 10 Jul 2013 14:11:39 +0200
Subject: drm: kill dev->ctx_start and dev->lck_start

Again completely unused, so just remove it.

Signed-off-by: Daniel Vetter <daniel.vetter@ffwll.ch>
Signed-off-by: Dave Airlie <airlied@gmail.com>
(cherry picked from commit c7e00b6d6a08772fac43b0fcea7fb48e6a1fe390)
Signed-off-by: Darren Hart <dvhart@linux.intel.com>
---
 drivers/gpu/drm/drm_fops.c | 3 ---
 include/drm/drmP.h         | 2 --
 2 files changed, 5 deletions(-)

diff --git a/drivers/gpu/drm/drm_fops.c b/drivers/gpu/drm/drm_fops.c
index 2f3b6fcaec0b..cdda3951ff03 100644
--- a/drivers/gpu/drm/drm_fops.c
+++ b/drivers/gpu/drm/drm_fops.c
@@ -80,9 +80,6 @@ static int drm_setup(struct drm_device * dev)
 	dev->last_checked = 0;
 	dev->if_version = 0;
 
-	dev->ctx_start = 0;
-	dev->lck_start = 0;
-
 	dev->buf_async = NULL;
 	init_waitqueue_head(&dev->buf_readers);
 	init_waitqueue_head(&dev->buf_writers);
diff --git a/include/drm/drmP.h b/include/drm/drmP.h
index bc802174f3e5..952478cd3ec3 100644
--- a/include/drm/drmP.h
+++ b/include/drm/drmP.h
@@ -1169,8 +1169,6 @@ struct drm_device {
 	spinlock_t event_lock;
 
 	/*@} */
-	cycles_t ctx_start;
-	cycles_t lck_start;
 
 	struct fasync_struct *buf_async;/**< Processes waiting for SIGIO */
 	wait_queue_head_t buf_readers;	/**< Processes waiting to read */
-- 
1.8.5.rc3

